verilog/parser/verilog.y

```diff
@@ -5479,6 +5479,18 @@ continuous_assign
     { $$ = MakeTaggedNode(N::kContinuousAssign, $1, $2, $3, $4, nullptr); }
   ;
 
+net_alias_assign
+  : lpvalue '=' net_alias_assign
```

Comment:
To treat this like other lists, I would flip the associativity like:
`: net_alias_assign '=' lpvalue`

Consider naming this rule `net_alias_assign_list` or `net_alias_lvalue_list`.

Reply:
Sure, renamed